lib/retry.js

```diff
@@ -142,7 +144,8 @@ function spawn(command, args) {
     debug('Command', command, args);
     return function() {
         return Q.Promise(function() {
-            var child = childProcess.spawn(command, args);
+            // var child = childProcess.crossSpawn(command, args);
+            var child = child.crossSpawn(command, args);
```

Comment:
Running this gives me `Error: TypeError: Cannot read property 'crossSpawn' of undefined`

Should this be `crossSpawn(command, args);` instead of `child.crossSpawn(command, args)`